api/src/java/org/sakaiproject/archiver/util/Dateifier.java

```diff
@@ -11,6 +11,10 @@
  */
 public class Dateifier {
 
+	private Dateifier() {
+		throw new IllegalArgumentException("Utility class");
```

Comment:
dont need the throw because if its private then no one can call it. just make it an empty constructor